doc: BT library docs clean up (#10954)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/enocean.rst

Commissioning
*************

> The devices must be commisioned to enable the library to process sensor or button messages from nearby EnOcean devices.

Reviewer: ```suggestion
The devices must be commissioned so that the library can process sensor or button messages from nearby EnOcean devices.
```
Typo in "commissioned", and I think this structure is better than the "to.. to.."

By default, the commissioning behavior may be triggered by interacting with the devices, as explained in the following sections.

To enable radio-based commissioning in your application, call the :c:func:`bt_enocean_commissioning_enable` function.

Commissioning wall switches
---------------------------

To set the EnOcean Easyfit wall switches into the commissioning mode, complete the following sequence:

1. Press and hold one of the buttons for more than seven seconds before releasing it.
#. Quickly press and release the same button (hold for less than two seconds).
#. Press and hold the same button again for more than seven seconds before releasing it.

> The switch transmits its security key that the library picks up and stores before calling the :c:member:`bt_enocean_callbacks.commissioned` callback.

Reviewer: ```suggestion
The switch transmits its security key, and the library picks up and stores the key before calling the :c:member:`bt_enocean_callbacks.commissioned` callback.
```
